src/bin/tikv-ctl.rs

```diff
@@ -353,21 +368,91 @@ fn dump_region_info(db: &DB, region_id: u64, skip_tombstone: bool) {
     println!("apply state: {:?}", apply_state);
 }
 
-fn dump_all_region_info(db: DB, skip_tombstone: bool) {
+fn convert_gbmb(mut bytes: u64) -> String {
+    let mb = if bytes % (1024 * 1024 * 1024) == 0 {
```

Comment:
Size humanrize is very common in tikv code base.
For future improvements, https://github.com/Thomasdezeeuw/human-size-rs/blob/master/src/lib.rs is a good implementation.